index.js

```diff
@@ -714,6 +714,23 @@
   //. 15
   //. ```
 
+  //# Maybe#sequence :: Applicative f => Maybe (f a) ~> (b -> f b) -> f (Maybe a)
+  //.
+  //. Evaluates an applicative action contained within the Maybe, resulting in:
+  //.
+  //.   - an applicative of `Just` the value of the evaluated action if `this`
+  //.     is a `Just`.
+  //.
+  //.   - a pure applicative of `Nothing` if `this` is a `Nothing`.
```

Comment:
It would be nice to mirror the structure of the description of `Maybe#reduce`:

> Evaluates an applicative action contained within the Maybe, resulting in:
>
>   - a pure applicative of a Nothing if `this` is a Nothing; otherwise
>
>   - an applicative of Just the value of the evaluated action.